Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM hexpm/elixir:1.11.4-erlang-24.0-rc3-alpine-3.13.3 AS build
+
+RUN apk add --no-cache build-base git python3
+
+WORKDIR /app
+
+# Install hex and rebar
+RUN mix local.hex --force && \
+    mix local.rebar --force
+
+# Build for production
+ENV MIX_ENV=prod
+
+# Install mix dependencies
+COPY mix.exs mix.lock ./
+COPY config config
+RUN mix do deps.get, deps.compile
+
+# Compile and build release
+COPY priv priv
+COPY lib lib
+# We need README.md during compilation
+# (look for @external_resource "README.md")
+COPY README.md README.md
+RUN mix do compile, release
+
+FROM hexpm/elixir:1.11.4-erlang-24.0-rc3-alpine-3.13.3
```

Comment:
Should we use Elixir 1.12 RC so we have Mix.install?

Reply:
So I thought, but I couldn't find that version on DockerHub, is there one?

Reply:
Ohhhh there’s probably not one, I’ll look into that soon

Reply:
we should get the RCs out sometime today, stay tuned!

Reply:
Lovely :rocket:

Reply:
Thank you! :cat: